back/engines/free/email_campaigns/spec/mailers/previews/email_campaigns/comment_on_idea_you_follow_mailer_preview.rb

```diff
@@ -5,23 +5,10 @@ class CommentOnIdeaYouFollowMailerPreview < ActionMailer::Preview
     include EmailCampaigns::MailerPreviewRecipient
 
     def campaign_mail
```

Comment:
I wonder if we could have moved the previews to `lib/mailer_previews` ([the Rails docs give this as an example](https://guides.rubyonrails.org/action_mailer_basics.html#previewing-emails)).